selenium/src/main/java/org/exoplatform/platform/qa/ui/selenium/locator/onlyOffice/OnlyOfficeLocator.java

```diff
@@ -1,11 +1,25 @@
 package org.exoplatform.platform.qa.ui.selenium.locator.onlyOffice;
 
 import com.codeborne.selenide.SelenideElement;
+import org.openqa.selenium.By;
 import static com.codeborne.selenide.Selenide.$;
 
 public class OnlyOfficeLocator {
 
     public static final SelenideElement ELEMENT_EDIT_ONlINE_BUTTON = $(".uiIconEcmsOnlyofficeOpen.uiIconEcmsLightGray.uiIconEdit");
     public static final SelenideElement ELEMENT_EDIT_ONLINE_BUTTON_FROM_PREVIEW = $(".onlyOfficeEditBtn");
+    public static final SelenideElement ELEMENT_DOWNLOAD_BUTTON_FROM_AS_TEXT = $(By.xpath("((//*[@class='pull-left statusAction'])[1]//a)[1]"));
+    public static final SelenideElement ELEMENT_OPEN_BUTTON_FROM_AS_TEXT = $(By.xpath("((//*[@class='pull-left statusAction'])[1]//a)[2]"));
```

Comment:
No need to check the buttons Download and Open IMO, there are not part of the feature. We should only check if the Edit Online button is displayed.

Reply:
In US3 I should check the alignment of Edit Online button with other buttons!! so I should compare edit online to the other buttons

Reply:
You do not have to check the alignment, only the existence (and maybe the order).

Reply:
US3 :
The "edit online" button should be aligned with other buttons in different locations:
-AS
-documents app
-file's preview